package.json

```diff
@@ -124,15 +130,15 @@
     "mocha": "^8.2.1",
     "pre-commit": "^1.1.3",
     "prettier": "^2.2.1",
-    "react": "^16.0.0",
-    "react-dom": "^16.0.0",
+    "react": "^16.14.0",
+    "react-dom": "^16.14.0",
     "react-router-dom": "^5.1.2",
     "react-test-renderer": "^16.12.0",
     "rimraf": "^3.0.2",
     "sinon": "^9.2.2",
     "terser-webpack-plugin": "^5.1.1",
     "ts-loader": "^8.0.12",
-    "typescript": "^4.1.3",
+    "typescript": "^4.9.4",
```

Comment:
It might be nicer to do this upgrade in a separate PR, to reduce the blast radius of this PR.

Reply:
As far as I understand, we do not need to upgrade typescript for jest, right?

Reply:
I agree, but there are some unfortunate errors when installing jest-environment-jsdom that fail without a newer TS version (not sure what the exact version is).

Might have to use an older version of jest in general to make everything work. Will try that.